app/src/main/kotlin/github/com/rhacco/dota2androidapp/lists/LiveMatchesListAdapter.kt

```diff
@@ -0,0 +1,48 @@
+package github.com.rhacco.dota2androidapp.lists
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.BaseAdapter
+import android.widget.TextView
+import github.com.rhacco.dota2androidapp.R
+
+class LiveMatchesListAdapter(context: Context) : BaseAdapter() {
```

Comment:
We should be using RecyclerView.Adapter, not BaseAdapter. Recyclerview already implements this recycling/viewholding pattern for reuse.

Reply:
Done